ledger-core/virtual/integration/method_test.go

```diff
@@ -348,867 +390,540 @@ func TestVirtual_CallMethodAfterPulseChange(t *testing.T) {
-
-	typedChecker := server.PublisherMock.SetTypedChecker(ctx, mc, server)
-	typedChecker.VCallRequest.SetResend(true).ExpectedCount(1)
-	typedChecker.VCallResult.Set(func(res *payload.VCallResult) bool {
-		switch res.Caller {
-		case objectAGlobal:
-			require.Equal(t, []byte("finish A.Bar"), res.ReturnArguments)
-		default:
-			require.Equal(t, []byte("finish A.Foo"), res.ReturnArguments)
-		}
-		// we should resend that message only if it's CallResult from A to A
-		return res.Caller == objectAGlobal
-	}).ExpectedCount(2)
-
-	pl := payload.VCallRequest{
-		CallType:            payload.CTMethod,
-		CallFlags:           payload.BuildCallFlags(contract.CallTolerable, contract.CallDirty),
-		Caller:              server.GlobalCaller(),
-		Callee:              objectAGlobal,
-		CallSiteDeclaration: class,
-		CallSiteMethod:      "Foo",
-		CallOutgoing:        server.RandomLocalWithPulse(),
-		Arguments:           insolar.MustSerialize([]interface{}{}),
-	}
-
-	beforeCount := server.PublisherMock.GetCount()
-	server.SendPayload(ctx, &pl)
-	if !server.PublisherMock.WaitCount(beforeCount+3, 10*time.Second) {
-		t.Fatal("failed to wait until all messages returned")
-	}
-
-	server.WaitActiveThenIdleConveyor()
-	mc.Finish()
-}
-
-// unordered A.Foo calls unordered A.Bar
-func TestVirtual_Call_UnorderedMethod_From_UnorderedMethod(t *testing.T) {
-	t.Log("C5122")
-
-	mc := minimock.NewController(t)
-
-	server, ctx := utils.NewUninitializedServer(nil, t)
-	defer server.Stop()
-
-	logger := inslogger.FromContext(ctx)
-
-	runnerMock := logicless.NewServiceMock(ctx, mc, func(execution execution.Context) string {
-		return execution.Request.CallSiteMethod
-	})
-	server.ReplaceRunner(runnerMock)
-	server.Init(ctx)
-	server.IncrementPulseAndWaitIdle(ctx)
-
-	var (
-		flags         = contract.MethodIsolation{Interference: contract.CallIntolerable, State: contract.CallDirty}
-		class         = gen.UniqueGlobalRef()
-		objectAGlobal = reference.NewSelf(server.RandomLocalWithPulse())
-	)
-
-	Method_PrepareObject(ctx, server, payload.Ready, objectAGlobal)
-
-	outgoingCall := execution.NewRPCBuilder(gen.UniqueGlobalRef(), objectAGlobal).CallMethod(objectAGlobal, class, "Bar", []byte{})
-	objectAExecutionMock := runnerMock.AddExecutionMock("Foo")
-	objectAExecutionMock.AddStart(
-		func(ctx execution.Context) {
-			logger.Debug("ExecutionStart [A.Foo]")
-		},
-		&execution.Update{
-			Type:     execution.OutgoingCall,
-			Error:    nil,
-			Outgoing: outgoingCall,
-		},
-	)
-	objectAExecutionMock.AddContinue(
-		func(result []byte) {
-			logger.Debug("ExecutionContinue [A.Foo]")
-		},
-		&execution.Update{
-			Type:   execution.Done,
-			Result: requestresult.New([]byte("finish A.Foo"), objectAGlobal),
-		},
-	)
-
-	runnerMock.AddExecutionMock("Bar").AddStart(
-		func(ctx execution.Context) {
-			logger.Debug("ExecutionStart [A.Bar]")
+// unordered A.Foo calls unordered B.Bar
```

Comment:
It's not useful now

Reply:
why? you add case in testrail and we have task PLAT-555 for realisation case

Reply:
because you have test names in table

Reply:
ok